### PR TITLE
MAINT-26328 : Feature/news: Update activity composer extension

### PR DESCRIPTION
**Pull request by aelbenney**
In order to make it possible for plugins to make rest calls, modifications,.. before self init, we added an event listener `activity-composer-extension-updated` to update the composer extension whenever needed.

### CODE REVIEW

File: webapp/vue-portlet/pom.xml [hunk 1]
```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (boubaker) [inline webapp/vue-portlet/pom.xml]: Please pay attention, this file is not needed anymore

**Review by ahamdi — APPROVED** [review]
LGTM

**Review by ahamdi — CHANGES_REQUESTED** [review]
Please open a task on maintenance.

**Review by ahamdi — APPROVED** [review]
LGTM

**Pull request merged**